README.md

```diff
@@ -55,7 +55,7 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 
-- [Live Demo Link](N/A)
+- [Live Demo Link](https://branbayou.github.io/leaderboard-list/dist)
 
```

Comment:
Hi, Bran, You did a great job 👍. I really appreciate your work. But It would be better if you design your ui correctly. When I run your code with a small screen it won't properly and break. Kindly fix it.